source/nvda.pyw

```diff
@@ -77,6 +77,15 @@ import logHandler
 from logHandler import log
 import winUser
 import winKernel
+# Avoid a E402 'module level import not at top of file' warning,
+# because monkeypatches need to be applied first first

```

Comment:
Is the second "first" a mistake, or have you added it just for emphasis?

Reply:
That was definitely a mistake.